# Goal
bool relative
float32 x
float32 y
float32 z
string target_frame
---
# Result
bool success
string message
float32 final_x
float32 final_y
float32 final_z
---
# Feedback
float32 x_position
float32 y_position
float32 z_position
bool moving
